packages/core/src/text/Display.tsx

```diff
@@ -1,21 +1,12 @@
-import { makePrefixer } from "../utils";
-import { clsx } from "clsx";
 import { forwardRef } from "react";
 import { Text, TextProps } from "./Text";
 
-const withBaseName = makePrefixer("saltText");
-
 export const Display1 = forwardRef<
   HTMLSpanElement,
   Omit<TextProps<"span">, "as">
 >(function Display1({ children, className, ...rest }, ref) {
   return (
-    <Text
-      as="span"
-      className={clsx(className, withBaseName(`display1`))}
-      ref={ref}
-      {...rest}
-    >
+    <Text as="span" styleAs="display1" ref={ref} {...rest}>
```

Comment:
```suggestion
    <Text as="span" styleAs={styleAs} ref={ref} {...rest}>
```

along with above change, i think this is slightly better as could be misleading otherwise